binr/radare2/radare2.c

```diff
@@ -855,6 +852,11 @@ int main(int argc, char **argv, char **envp) {
 
 	r_bin_force_plugin (r.bin, forcebin);
 
+	if (r_config_get (r.config, "prj.name")) {
```

Comment:
dont get this var twice, just save in a local var and then use it. but this will be always true. because it will be "", not NULL, so you have to check for (*prj_name) instead